### DISCUSSION TITLE
node: bad option: --experimental-wasm-eh

### DISCUSSION THREAD

**Discussion by ejohn-cricut** [cricut/FishyJoes#159] [Q&A]
typescript / wasm testing step fails with error: `node: bad option: --experimental-wasm-eh`

**Comment by ejohn-cricut** [comment]
Most likely an installed node version issue, to workaround, temporarily remove the flag from `node-test/package.json` in the bindings repo.

**Reply by kellyo-cricut** [reply]
Do we have a formally approved node version? Just match what the CI does?
